TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/datahandlers/http/HttpJsonTransformingConsumerTest.java

```diff
@@ -118,7 +132,7 @@ private void walkMaps(Object o) {
         try (var sampleStream = HttpJsonTransformingConsumer.class.getResourceAsStream(
                 "/requests/raw/post_formUrlEncoded_withFixedLength.txt")) {
             var allBytes = sampleStream.readAllBytes();
-            Arrays.fill(allBytes, allBytes.length-10, allBytes.length, (byte)' ');
+            Arrays.fill(allBytes, allBytes.length-30, allBytes.length, (byte)' ');
```

Comment:
This change was needed otherwise netty converted the request instead of parsing in the HttpServerCodec/HttpObjectDecoder. Created a separate item to evaluate behavior and correct if needed https://opensearch.atlassian.net/browse/MIGRATIONS-1647